build/wflow_cli/test/runtests.jl

```diff
@@ -96,3 +80,19 @@ end
         @test filesize(joinpath(outputdir, file)) > 0
     end
 end
+
+@testset "wflow_sbm_timing" begin
+
+    ENV["JULIA_NUM_THREADS"] = 1
```

Comment:
Probably better to set the environment locally as shown here:
https://docs.julialang.org/en/v1/manual/running-external-programs/#Cmd-Objects

Reply:
Thanks for this suggestion, this is indeed a better way to do this!